R/call.R

```diff
@@ -25,11 +25,178 @@ call_new <- function(f, ..., .args = list()) {
   as.call(c(f, args))
 }
 
+#' Standardise a call against formal arguments
+#'
+#' Compared to \code{\link{match.call}()}, \code{call_standardise()}:
+#' \itemize{
+#'   \item Always report dotted arguments as such: \code{call(..1, ..2)}.
+#'         In comparison, \code{match.call()} inlines literals:
+#'         \code{call("foo", 10)}.
+#'   \item Provides an argument \code{enum_dots} to impose
+#'         enumerated names on dotted arguments. This produces
+#'         \code{call(..1 = x, ..2 = ..3)} instead of
+#'         \code{call(foo = x, ..3)}.
+#'   \item Does not sort arguments according to the order of
+#'         appearance in the function definition.
+#'   \item Does not partial-match arguments.
```

Comment:
oh right, this could create problems for arg introspection if we don't partial match.